pom.xml

```diff
@@ -89,6 +87,12 @@ Brotli, Zstandard and ar, cpio, jar, tar, zip, dump, 7z, arj.
     <checkstyle.config.file>${basedir}/src/conf/checkstyle/checkstyle.xml</checkstyle.config.file>
     <checkstyle.suppress.file>${basedir}/src/conf/checkstyle/checkstyle-suppressions.xml</checkstyle.suppress.file>
     <checkstyle.resourceExcludes>LICENSE.txt, NOTICE.txt, **/maven-archiver/pom.properties</checkstyle.resourceExcludes>
+    <!--
```

Comment:
There is no need to make this PR noisier by moving comments and changing their format to whatever this "~" is.

Reply:
I moved all the overwritten versions together, so that we can more easily check each of those overrides, when we upgrade `commons-parent`.

I minimized the changes in https://github.com/apache/commons-compress/pull/712/commits/adcb46b6e785fac63153f3bb604c831d9f0bb37e